catboost/private/libs/algo/tensor_search_helpers.cpp

```diff
@@ -20,9 +20,9 @@ using namespace NCB;
 
 
 TSplit TCandidateInfo::GetBestSplit(
-    const TTrainingDataProviders& data,
-    const TFold& fold,
-    ui32 oneHotMaxSize
+        const TTrainingDataProviders& data,
```

Comment:
please restore indentation -- once this is done, i will get back with more comments